scripts/getStatus.js

```diff
@@ -50,11 +52,10 @@ function writeContent(content) {
   fs.writeFileSync(logFilepath, content);
 }
 
-//runs the linter on a given swagger spec.
-async function runLinter(readme) {
-    let cmd = 'autorest ' + readme + ' --azure-validator=true --message-format=json';
+//runs the command on a given swagger spec.
+async function runCmd(linterCmd) {
+    let cmd = linterCmd;
```

Comment:
Is this needed?

Reply:
Changed to remove linterCmd and use cmd only.

Reply:
Fixed...thanks.